psalm.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="2"
+    resolveFromConfigFile="true"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <projectFiles>
+        <directory name="src" />
```

Comment:
Might want to add `<directory name="tests" />` but that would mean more issues to be fixed, suppressed, or baselined.

Reply:
Let's do `src` only for now, you can still propose another PR for `tests` later on!